__tests__/api.test.js

```diff
@@ -31,4 +32,13 @@ describe.only("GET /api/topics", () => {
   test("404: responds with an error message saying that the requested resource does not exist", () => {
     return request(app).get("/api/topicss").expect(404);
   });
+  test("200: responds with an an object of available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual(endpointData);
+      });
```

Comment:
Very tidy 👍 
A reminder, if you wanted to, you could destructure the body in the call back of the then block like so ```then(({body})=>{})```